### ISSUE TITLE
Bakery setup wizard for SMTP config + separate SMTP  setup in it's own command

### ISSUE THREAD

**Issue by lcharette** [userfrosting/UserFrosting#874]
[To quote @alexweissman](https://chat.userfrosting.com/group/dev?msg=EfyF6HsSQfmQNLLZa)

> could do some kind of wizard with Gmail, Mailgun, Other as options. the same way we do for DB setup in Bakery

To help with recurrent SMTP issues (See #552 & #776) 

Simply need to add a selection when creating SMTP server : 
```
1) SMTP
2) Gmail
3) Mailgun
4) sendmail
5) No email support / later --> big warning registration won't work
``` 

And ask pertinent question when dealing with gmail for example. 

To sync with #872.

Also, we should separate `setup` command into `setup:db`  and `setup:smtp`

**Comment by lcharette** [comment]
This was planed for 4.2, but finally been added to hotfix for 4.1.18 release. Requires a `composer update`.

**Comment by alexweissman** [comment]
Released in 4.1.18.
